scripts/extractCodeSnippets.js

```diff
@@ -24,7 +24,14 @@ function findAllCodeSnippetsInJsDoc(jsDoc) {
       if (!codeSnippets.has(operationId)) {
         codeSnippets.set(operationId, []);
       }
-      codeSnippets.get(operationId).push(stripMarkdownCodeSnippet(rawCode));
+
+      try {
+        codeSnippets.get(operationId).push(stripMarkdownCodeSnippet(rawCode));
+      } catch (e) {
+        console.error(
+          `Error in operation ${operationId} with code snippet: ${rawCode}, value: ${value}`
+        );
+      }
```

Comment:
```suggestion
      } catch (e) {
        console.error(
          `Error in operation ${operationId} with code snippet: ${rawCode}, value: ${value}`
        );
        throw e;
      }
```
I would throw the exception here, to not ignore the error. Now, we are giving more context about the problem, making it easier to debug.